### ISSUE TITLE
Linter errors obscure full screen

### ISSUE THREAD

**Issue by zapman449** [atom-community/linter#764]
[linter obscuring full screen with errors]

![Screenshot or GIF movie](https://dl.dropboxusercontent.com/u/5493620/screen_full_of_lint_errors.png)

## Repro Steps

1. write valid python code
2. early in code, add open paren 
3. see screen full of errors (see shot) and unable to continue editing.

**Expected:** Perhaps truncate a long list of errors so it doesn't obscure the full screen
**Actual:** editing a file blind is challenging.

## Command History

```
     -0:00.0 bug-report:open (atom-workspace.workspace.scrollbars-visible-when-scrolling.theme-one-dark-syntax.theme-one-dark-ui)
```

## Versions

* **Atom:**       1.0.2
* **Atom-Shell:** 0.22.3
* **OS:**         Mac OS X 10.10.4
* **Misc**
    * apm  1.0.1
    * npm  2.5.1
    * node 0.10.35
    * python 2.7.8
    * git 2.1.2
    * linter 1.2.4
    * linter-flake8 1.5.0

---

<small>This report was created in and posted from the Atom editor using the package `bug-report` v0.7.1.</small>

**Comment by steelbrain** [comment]
Dupe of https://github.com/atom-community/linter/issues/760
and https://github.com/atom-community/linter/issues/746 and https://github.com/atom-community/linter/issues/723 and https://github.com/atom-community/linter/issues/709
